agreements/membership/egc-representative-agreement.yaml

```diff
@@ -0,0 +1,29 @@
+title: "Extended Governance Committee (EGC) Membership Agreement"
+version: "initial draft"
+preamble: |
+  As a project representative to the Extended Governance Committee (EGC), I affirm the following:
+
+attestations:
+  bylaws:
+    title: "Commonhaus Foundation Bylaws"
+    body: |
+      I have read the following sections of the Commonhaus Foundation Bylaws and understand my obligations:
+
+      - [Project Representation](https://www.commonhaus.org/bylaws/2-cf-membership.md#project-representation)
+      - [Composition of the EGC](https://www.commonhaus.org/bylaws/3-cf-council.md#extended-governance-committee-egc)
+      - [Decision Making](https://www.commonhaus.org/bylaws/decision-making.html)
+      - [Matters requiring EGC approval](https://www.commonhaus.org/bylaws/3-cf-council.md#matters-requiring-egc-approval)
+      - [Electronic Participation and Voting](https://www.commonhaus.org/bylaws/cf-council.html#electronic-participation-and-decision-making)
+
+  coc:
+    title: "Code of Conduct"
+    body: |
+      I have read the [Code of Conduct](https://www.commonhaus.org/policies/code-of-conduct/) and agree to adhere to its terms.
+
+  email:
+    title: "Commonhaus email address"
+    body: |
+      I have read the [Forward Email Terms of Service](https://forwardemail.net/en/terms) and agree to adhere to its terms when using my `commonhaus.dev` email address.
+
+      > [!NOTE]
+      > If you accept these terms, a `commonhaus.dev` email address will be created for your GitHub Login if it does not already exist. It will initially forward to the public address listed in your GitHub profile.
```

Comment:
This forces potential members to have a GitHub account regardless of their stance of FLOSS code hosting solutions. Some developers object to GitHub and prefer GitLab. 

I think it may be important to distinguish between just having a GH account to participate in CF discussions and votes vs. a GH for hosting/working on project code.

Reply:
Fair point. Given we're managing discussions and votes on GH, I would hope that much should be obvious. Will need to double-check project on-boarding materials to make sure it is clear that project code can be hosted elsewhere. Foundation business, decision-making, and record-keeping, however, is here. ;)

Reply:
I updated the agreement text here, and also added language to #154  .. does that help?